### PR TITLE
Add Intel Xe Super Sampling

### PR DESCRIPTION
**Pull request by Lifeismana**
### SteamDB app page links to a few games using this
https://steamdb.info/app/835570/
https://steamdb.info/app/950410/
https://steamdb.info/app/1399080/
https://steamdb.info/app/1824220/
### Brief explanation of the change
Add Intel Xe Super Sampling
I went with looking for `igxess.dll` and `libxess.dll` but the couple of apps that are shipping it all have the following dlls:
```
XeFX.dll
XeFX_Loader.dll
dxcompiler.dll
dxil.dll
igxess.dll
libxess.dll
```
so it might be worth it to take a look at it later when more apps are shipping it to see if that choice was a good one

### CODE REVIEW

File: rules.ini [hunk 1]
```diff
@@ -147,6 +147,7 @@ cURL = curl(?:module|lib|d|-4|-ttv|-x64|64|_pluginw64_release|)\.(?:dll|exe|lib|
 Discord = (?:^|/)(?:lib)?discord(?:|-rpc|_game_sdk)\.(?:dll|dylib|so)$
 EpicOnlineServices = (?:^|/)(?:lib)?eossdk
 FMOD = (?:^|/)(?:lib)?fmod(?:l|ex|exl|studio|studiol|)(?:64)?\.(?:dylib|dll|so)$
+Intel_XeSS = xess\.dll$
```
Comment (xPaw) [inline rules.ini:150]: This will match anythingxes.dll, why?

is ig|lib not enough?

Comment (Lifeismana) [reply rules.ini:150]: fair, i was just lazy

Comment (xPaw) [reply rules.ini]: Add (?:^|/)

Comment (Lifeismana) [reply rules.ini]: knew i forgot something

**Pull request merged**